### PR TITLE
Task 5 Brain MRI Super Resolution Network - Yufan Wang s45533675

### PR DESCRIPTION
**Pull request by WYF0001**
Yufan Wang s45533675
Data: ADNI MRI Dataset
dataset.py: Define functions to load data from the given path, resize images and assign in batch, divide training set and validation set, and do normalization.
modules.py: Define functions to get model, down-sampling, up-scaling, and plot result images.
train.py: Define functions for calculating PSNR, ModelCheckpoint and EarlyStopping callbacks. And train the network.
predict.py: Use test images to evaluate the network.

### CODE REVIEW

**Comment by uqmarlonbran** [comment]
Performance seems in line with most other implementations. All seems ok.

**Comment by shakes76** [comment]
## Good Practice (Design/Commenting, TF/Torch Usage)
Adequate use and implementation
Good spacing and comments
Header blocks missing -1
## Recognition Problem 
Solves problem
Driver Script present
File structure present
Shows Usage & Demo & Visualisation & Data usage (no plots) -1
Module present
Commenting (more could be done) -1
No Data leakage
Difficulty: Normal -5
## Commit Log
Meaningful commit messages
Progressive commits used
## Documentation
ReadMe acceptable, needs more info -1
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Feedback required
Request Description good
